content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,31 @@
+---
+pim_announcement_img: ./img/api-filter-attributes.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
+::: meta-data type="New" features="API, Connectivity" available="late August" link-to-doc="https://api.akeneo.com/documentation/filter.html"
+:::
+
+With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.
+For example, you can retrieve data for all attributes that have been updated since a specific date, or you can get the data from all the sub-categories of a specific tree.
+
+::: more
+[How to filter attributes?](https://api.akeneo.com/documentation/filter.html#filter-attributes)
+[How to filter attribute groups?](https://api.akeneo.com/documentation/filter.html#filter-attribute-groups)
+[How to filter families?](https://api.akeneo.com/documentation/filter.html#filter-families)
+[How to filter categories?](https://api.akeneo.com/documentation/filter.html#filter-categories)
+:::
+
+# Option labels are now available in the product endpoint response
```

Comment:
It should be in a separate file. One feature, one file. Otherwise, it won't appear in the PIM.
If the two features are too small to be advertised separately, you can group them in the same file, but you'll need to have only one level-one title with inside two subsections with level-two titles. Don't hesitate to ping me if not clear.